puppet/modules/koji/files/mirror_reposync.cron

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Should this use `-e` so we know when it fails or does cron do that automatically?

Reply:
cron does automatically if there is output, we're using `--quiet` to `dnf`, so by default it will be silent, but I'd expect output when there are errors -- will test

Reply:
```console
[root@mirror01 ~]# dnf --config reposync.conf reposync --download-path=/srv/mirror --download-metadata --quiet
[root@mirror01 ~]# vim reposync.conf 
[root@mirror01 ~]# dnf --config reposync.conf reposync --download-path=/srv/mirror --download-metadata --quiet
Error: Failed to download metadata for repo 'codeready-builder-for-rhel-9-x86_64-rpms': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
[root@mirror01 ~]# echo $?
1
```

IMHO sufficient?